peekaboo/ruleset/__init__.py

```diff
@@ -38,6 +38,21 @@ class Result(Enum):
     failed = 5
     bad = 6
 
+    def __eq(self, other):
```

Comment:
Why? We should either have an outlook what it's going to be used for in this commit's message or it should perhaps live in the commit where it's usage is introduced as well (still with an explanation why for my taste).